pystruct/learners/n_slack_ssvm.py

```diff
@@ -12,13 +12,17 @@
 import cvxopt
 import cvxopt.solvers
 
-from sklearn.externals.joblib import Parallel, delayed
+from sklearn.externals.joblib import Parallel, delayed, cpu_count
 from sklearn.utils import gen_even_slices
 
 from .ssvm import BaseSSVM
 from ..utils import unwrap_pairwise, find_constraint
 
 
+def find_constraint_map(args):
+    return find_constraint(* args)
```

Comment:
Yes that's a good idea. Could also pass an optional pool. Cleaner than wrapping functions. Where do you think it should go? A new general utils file?